fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/mv/MaterializedViewRule.java

```diff
@@ -236,11 +250,17 @@ private void collectGroupByAndAggFunctions(OptExpression root) {
         }
 
         Operator operator = root.getOp();
+        if (operator instanceof LogicalOlapScanOperator) {
+            LogicalOlapScanOperator scanOperator = (LogicalOlapScanOperator) operator;
+            traceScanIds.add(factory.getRelationId(scanOperator.getOutputColumns().get(0).getId()));
+        }
+
```

Comment:
Why not in collectScanOutputColumns()?

Reply:
collectGroupByAndAggFunctions and  collectScanOutputColumns  both access the entire plan from the bottom up， traceScanIds is used for record relationId below the aggregate node